src/status_im2/contexts/chat/composer/link_preview/style.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im2.contexts.chat.composer.link-preview.style)
+
+(def padding-horizontal 20)
+(def preview-list-padding-top 12)
+(def preview-list-padding-bottom 8)
+(def preview-height 56)
```

Comment:
Please move the `preview-height` to the constants file.

Reply:
I intentionally put it here, because it's used by the style namespace and also to build this map in another file `:container-style-item {:height style/preview-height}`. I think it's more style related than anything else.

But since you made a comment I'll add it there.

Reply:
Solved. I moved `preview-total-height` to the composer constants file, and I kept `preview-height` here because it's used by its related view file.

Reply:
> because it's used by the style namespace

@ilmotta I understand it's a style, but this is to keep all the components' height values grouped together in one file. Also, these values are not local to that sub-component, they could be needed by other comps in the composer. In the old composer, it was a mess.

Reply:
+1 @OmarBasem I think for common styles it is better to use a constants file because it is quite unexpected for a style file to be shared across several components/view files and likely that a developer will miss that and could break a lot of ui's. In general I think we should keep it as 1 style file per 1 view file and use this constants approach if some settings are reused across multiple files.